java/runtime/src/main/java/io/ray/runtime/object/LocalModeObjectStore.java

```diff
@@ -117,11 +117,11 @@ public Address getOwnerAddress(ObjectId id) {
   }
 
   @Override
-  public byte[] getOwnershipInfo(ObjectId objectId) {
-    return new byte[0];
+  public OwnershipInfo getOwnershipInfo(ObjectId objectId) {
+    return new OwnershipInfo(new byte[0], new byte[0]);
```

Comment:
Maybe returning null is easier to understand?

Reply:
Let me try and see the CI result. not sure if it will cause some NPE implicitly.

Reply:
It works.